pkg/api/validation/validation.go

```diff
@@ -176,6 +176,17 @@ func nameIsDNSSubdomain(name string, prefix bool) (bool, string) {
 	return false, dnsSubdomainErrorMsg
 }
 
+// nameIsDNS1123Label is a ValidateNameFunc for names that must be a DNS 1123 label.
+func nameIsDNS1123Label(name string, prefix bool) (bool, string) {
```

Comment:
I'll let this inconsistency slide.  I'm ambivalent about which is better, but it's inconsistent as-is.

nameIsDNSSubdomain vs nameIsDNS1123Label